src/views/Workflow.vue

```diff
@@ -39,6 +39,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             :workflows="tree.root.children"
           />
         </v-skeleton-loader>
+        <v-skeleton-loader
+            v-for="widgetId of tableWidgets"
+            :key="widgetId"
+            :id="widgetId"
+            :loading="isLoading"
+            type="list-item-three-line"
+            tab-title="table"
+        >
+         <table-component
+             :tasks="table"
+         />
+        </v-skeleton-loader>
```

Comment:
Can you re-format the code above to use 2 spaces instead of 4 like the rest of the code, please?

Reply:
Spaces removed.